src/mmw/js/src/data_catalog/models.js

```diff
@@ -1,45 +1,70 @@
 "use strict";
 
-var _ = require('underscore'),
+var $ = require('jquery'),
+    _ = require('underscore'),
     Backbone = require('../../shim/backbone'),
     turfIntersect = require('turf-intersect'),
     App = require('../app'),
+    settings = require('../core/settings'),
     utils = require('./utils');
 
 var REQUEST_TIMED_OUT_CODE = 408;
 var DESCRIPTION_MAX_LENGTH = 100;
+var PAGE_SIZE = settings.get('data_catalog_page_size');
 
 var Catalog = Backbone.Model.extend({
     defaults: {
         id: '',
         name: '',
         description: '',
+        query: '',
+        bbox: '',
         loading: false,
         active: false,
         results: null, // Results collection
         resultCount: 0,
+        page: 1,
         error: '',
     },
 
     search: function(query, bounds) {
-        var bbox = utils.formatBounds(bounds),
-            data = {
-                catalog: this.id,
-                query: query,
-                bbox: bbox
-            };
+        this.set({
+            query: query,
+            bbox: utils.formatBounds(bounds),
+        });
 
-        return this.startSearch(data)
+        return this.startSearch(1)
```

Comment:
Yeah. When accessing results in the URL, it does make more sense to ask for `page=2` for the second page than `page=1`. In this case, I think a 1-based index is more idiomatic.